frame/example/src/lib.rs

```diff
@@ -577,7 +577,7 @@ impl<T: Config> Module<T> {
 //
 // Note that a signed extension can also indicate that a particular data must be present in the
 // _signing payload_ of a transaction by providing an implementation for the `additional_signed`
-// method. This example will not cover this type of extension. See `CheckRuntime` in FRAME System
+// method. This example will not cover this type of extension. See `CheckSpecVersion` in [FRAME System](https://github.com/paritytech/substrate/tree/master/frame/system#signed-extensions)
```

Comment:
```suggestion
// method. This example will not cover this type of extension. See `CheckSpecVersion` in 
// [FRAME System](https://github.com/paritytech/substrate/tree/master/frame/system#signed-extensions)
```

Reply:
line width is 100, maximum 120. https://gitlab.parity.io/parity/substrate/-/jobs/804042